frontend/src/components/elements/DeckGlJsonChart/styled-components.ts

```diff
@@ -30,3 +30,10 @@ export const StyledDeckGlChart = styled.div<StyledDeckGlChartProps>(
     width,
   })
 )
+
+export const StyledNavigationControlDiv = styled.div(() => ({
+  position: "absolute",
+  right: 40,
```

Comment:
This seems like change in design? What's going on here?

Not required, but have we considered using theming here and use `theme.spacing.md` or `theme.spacing.sm`? It will add or subtract 2 px, but the benefit we get is consistent spacing that we can change with one variable.